### PR TITLE
Add jax.numpy.linalg functions to Jax frontend Update linalg.py

### PR DESCRIPTION
**Pull request by devsugun**
numpy.linalg.solve

### CODE REVIEW

**Comment by kimbring2** [comment]
@devsugun Hello, you should add the solve function under https://github.com/unifyai/ivy/blob/master/ivy/functional/frontends/jax/numpy/linalg.py.

Additionally, follow the form of other functions.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.
